release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -65,6 +65,16 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 * With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
 * With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
 
+[id="ocp-4-12-helm-page-improvements"]
+==== Helm page improvements
+
+In {product-title} 4.12, you can use the Helm page to:
+
+* Create Helm Releases and Repositories using the *Create* button.
+* Create, update, or delete a cluster-scoped or namespace-scoped Helm chart repository.
+* View the list of the existing Helm chart repositories in the *Repositories* tab.
```

Comment:
This can also include a newly added scope field
```suggestion
* View the list of the existing Helm chart repositories with their scope in the *Repositories* tab.
```

Reply:
Fixed.